src/engine/output-results.ts

```diff
@@ -133,22 +133,30 @@ function formatDocumentation(
   return resource;
 }
 
+/**
+ * Helper function to determine whether the action represented by
+ * the given resource has been "completed", and the pathway execution should advance.
+ * @param resource - the FHIR resource for a given action
+ * @return boolean as to whether this resource is complete
```

Comment:
I think you missed a dash here
```suggestion
 * @return boolean - as to whether this resource is complete
```